src/model/serialize_meta_details.rs

```diff
@@ -105,13 +106,21 @@ pub fn serialize_meta_details(
     let meta_item = meta_details
         .meta_items
         .iter()
-        .find(|meta_item| matches!(&meta_item.content, Some(Loadable::Ready(_))))
+        .find(|meta_item| {
+            if matches!(&meta_item.content, Some(Loadable::Ready(_))) {
+                tracing::info!("matched content: {:?}", meta_item.content);
```

Comment:
Why logging this ?

Reply:
yes, leftover from my testing. Good spotting.